presto-expressions/src/main/java/com/facebook/presto/expressions/LogicalRowExpressions.java

```diff
@@ -370,6 +379,101 @@ public RowExpression filterConjuncts(RowExpression expression, Predicate<RowExpr
         return combineConjuncts(conjuncts);
     }
 
+    public static GroupingElement removeGroupingElementPrefix(GroupingElement groupingElement, Optional<Identifier> removablePrefix)
```

Comment:
I do not find this appropriate as this would created dependency on the parser.

Reply:
Yeah I've raised my concern with Zhan. We are hoping James would have better idea where we can put these methods.

Reply:
May lets ping James in the group chat and get consensus?
I would move the changes to ExpressionUtils to start with and then make changes if required.